src/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.cs

```diff
@@ -7,7 +7,9 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
+#if !netstandard
```

Comment:
Is `netstandard` the appropriate constant to #ifdef on or should we use something like `#if netcoreapp3_0` instead?

Reply:
cc @terrajobst

Reply:
This is wrong, because we want to be exclusive with this and not inclusive be default. The reason is that we want the source package to not need to define any constants for building, and the way this is now, they would have to define netstandard constant in order for it to work.